include/rocksdb/db.h

```diff
@@ -1171,6 +1172,15 @@ class DB {
     return Status::NotSupported("PromoteL0() is not implemented.");
   }
 
+  // Trace DB operations. Use EndTrace() to stop tracing.
+  virtual Status StartTrace(const TraceOptions& /*options*/,
+                            std::unique_ptr<TraceWriter>&& /*trace_writer*/) {
+    return Status::NotSupported("StartTrace() is not implemented.");
+  }
+
+  virtual Status EndTrace(const TraceOptions& /*options*/) {
```

Comment:
Does EndTrace() needs TraceOptions too? And this is the same option as what is passed to StartTrace().

Reply:
I added TraceOptions here as I was anticipating certain options to change between Start and End. But on second thought, I agree with you that we might not ever need it. If there was ever such a need, we could easily add a default argument at that time. I'll remove TraceOptions from EndTrace for now.